test/e2e/healthcheck_test.go

```diff
@@ -0,0 +1,80 @@
+package e2e
+
+import (
+	"os"
+	"path/filepath"
+	"strings"
+	"time"
+
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+)
+
+var _ = Describe("When testing healthcheck [healthcheck]", func() {
```

Comment:
```suggestion
var _ = Describe("When testing healthcheck [healthcheck] [features]", func() {
```
If you'd like to run it with the feature tests, add this.

Reply:
This is resolves and decided to run healthcheck a part of remediation test.